docs/production/authentication/microsoft.mdx

```diff
@@ -181,6 +181,24 @@ You may need to restart your Mage server for the changes to take effect.
 
 </AccordionGroup>
 
+### Configurable environment variables
+To configure the environment for Microsoft Azure Cloud and Microsoft Azure Gov Cloud sign-in, 
+you need to set the following environment variables:
+1. You can specify the `AZURE_HOST` URLs based on your environment:
+```plaintext
```

Comment:
@colefreeman Is including the `plaintext` after the triple backticks intentional? What does the doc look like if we removed that?

Reply:
removed plain text and added 2. at line 194, tested and woring in minlify dev env.
<img width="791" alt="Screenshot 2024-08-22 at 6 21 42 AM" src="https://github.com/user-attachments/assets/55bf22fd-6a0f-452e-b337-549107c00c9e">

Reply:
I guess it doesn't make much of a difference